src/util/cmdlineargs.cpp

```diff
@@ -73,39 +75,54 @@ bool CmdlineArgs::parse(int argc, char** argv) {
     for (int a = 0; a < argc; ++a) {
         arguments << QString::fromLocal8Bit(argv[a]);
     }
+    return parse(arguments, false);
+}
+
+void CmdlineArgs::parseForUserFeedback() {
+    DEBUG_ASSERT(QCoreApplication::instance());
+    if (!m_hasUserFeedback) {
```

Comment:
This condition seems to be wrong or I misunderstand the purpose/naming of the boolean.

Reply:
We need only execute the second run when the first run has produced user feedback, otherwise we can skip the second run. I am happy to rename the variable if you have a suggestion.

Reply:
`m_requiresUserFeedback`?

Reply:
This may be read as if we require feedback from the user. 
How about m_parseForeUserFeedbackRequired?